data/campaigns/Heir_To_The_Throne/scenarios/17_Scepter_of_Fire.cfg

```diff
@@ -88,6 +90,9 @@
                     [/gold_carryover]
 
                     [note]
+                        [show_if]
+                            {VARIABLE_CONDITIONAL moved_too_close boolean_equals false}
+                        [/show_if]
```

Comment:
Will re-test and adjust if needed.

I'm considering, if changes are needed forgetting zookeeper's desire for separate PR for each feature and going back to an all-in-one. It's cleaner and a LOT less confusing.